test/run_test_apache.sh

```diff
@@ -32,6 +32,21 @@ if [ "$#" -gt 3 ] ; then
 else
   SSL="false"
 fi
+
+if [ "$#" -gt 4 ] && [[ $5 == "--local=true" ]] ; then
```

Comment:
see_no_evil

Reply:
Hm. Can we decouple parameters from their order? 
Like checking whether any parameter is `--local=true`, or even better, just `--local`.

Reply:
Changed to `--skipProxy` to make it more descriptive